### ISSUE TITLE
Schema validation error with property patterns (regex) when data is invalid

### ISSUE THREAD

**Issue by sadiqhabib** [josdejong/jsoneditor#682]
I am trying to use schema only for properties i.e `additionalProperties` because I don't have any predefined property. It throws error in console instead of showing in view.
Shema: 
```
{
  type: 'object',
  patternProperties: { '^[a-z0-9_-]+$': { 'type': ['number', 'string'] } },
  additionalProperties: false,
}
```
Working with data:
`{ foo: 1, bar: 3, baz: 'str' }`
Not Woking with data:
`{ foo: 1, bar: 3, baz: 'str', fer: true }`
Error: 

```
ERROR Error: Invalid JSON path: unexpected character "[" at index 0
    at Object.parsePath (util.js:811)
    at Node.push../node_modules/jsoneditor/src/js/Node.js.Node.findNode (Node.js:212)
    at findNode (treemode.js:595)
    at Array.map (<anonymous>)
    at JSONEditor.push../node_modules/jsoneditor/src/js/treemode.js.treemode.validate (treemode.js:593)
    at JSONEditor.push../node_modules/jsoneditor/src/js/treemode.js.treemode.set (treemode.js:216)
    at JSONEditor.push../node_modules/jsoneditor/src/js/JSONEditor.js.JSONEditor.setMode (JSONEditor.js:289)
    at JSONEditor.push../node_modules/jsoneditor/src/js/JSONEditor.js.JSONEditor._create (JSONEditor.js:198)
    at new JSONEditor (JSONEditor.js:148)
```

**Comment by josdejong** [comment]
Thanks for reporting @sadiqhabib 

Looks like JSONEdtior it tries to parse the regex as a JSON path when trying to find the node where to attach the validation error reported by Ajv.

Anyone interested looking into this issue?

**Comment by josdejong** [comment]
This should be fixed now in `v5.32.4`, see also #687.
